projects/sandbox/pyproject.toml

```diff
@@ -72,10 +73,8 @@ batches_per_epoch = 200
 max_epochs = 300
 
 lr = 4e-3
-min_lr = 1e-5
-decay_steps = 10000
-weight_decay = 1e-5
-early_stop = 50
+
+early_stop = 100
```

Comment:
why this change?

Reply:
To let the network train for longer. Feel free to change. Like I said before, the parameters that are version controlled are more or less irrelevant until we find a configuration that works

Reply:
OK. Fine with me.